pkgs/development/beam-modules/lib.nix

```diff
@@ -12,6 +12,15 @@ rec {
 
   callPackage = callPackageWith pkgs;
 
+  /* Erlang/OTP-specific version retrieval, returns 19 for OTP R19 */
+  getVersion = x:
+   let
+     parse = drv: (builtins.parseDrvName drv).version;
+   in builtins.replaceStrings ["B" "-"] ["." "."] (
+      if stdenv.lib.isString x
```

Comment:
I get this:

~/c/nixpkgs ❯❯❯ nix-build --show-trace -A beam.packages.erlangR18.elixir  /home/gleber/code/nixpkgs

error: while evaluating ‘versionAtLeast’ at /home/gleber/code/nixpkgs/lib/strings.nix:363:24, called from /home/gleber/code/nixpkgs/pkgs/development/beam-modules/default.nix:40:21:
while evaluating ‘versionOlder’ at /home/gleber/code/nixpkgs/lib/strings.nix:351:22, called from /home/gleber/code/nixpkgs/lib/strings.nix:363:29:
while evaluating ‘callPackageWith’ at /home/gleber/code/nixpkgs/lib/customisation.nix:95:35, called from /home/gleber/code/nixpkgs/pkgs/development/beam-modules/default.nix:6:9:
while evaluating ‘makeOverridable’ at /home/gleber/code/nixpkgs/lib/customisation.nix:54:24, called from /home/gleber/code/nixpkgs/lib/customisation.nix:99:8:
undefined variable ‘isString’ at /home/gleber/code/nixpkgs/pkgs/development/beam-modules/lib.nix:20:10